File: Python/list/pos.py
#WAP to store the positive and negative list element in 2 diff lists
li=[]
pos=[]
neg=[]
c=0
n=int(input("Enter the range = "))
for i in range(0,n):
    li.append(int(input("Enter the no.= ")))
print("List = ",li)
for i in li:
    if i>0:
        pos.append(i)
    elif i<0:
        neg.append(i)
    else:
        c+=1
print("Positive List = ",pos)
print("negative List = ",neg)
print("No. of zero = ",c)

File: C Programms/Switch Case/pos.c
//WAP to check positive no.
#include<stdio.h>
void main()
{
    int n;
    printf("Enter the no. = ");
    scanf("%d",&n);
    switch ((n>0)?1:0)
    {
    case 1:
        printf("%d is a positive no.",n);
        break;
    case 0:
        if (n==0)
            printf("%d is a neutral no.",n);
        else
            printf("%d is a negative no.",n);
        break;    
    default:
        break;
    }
}